backend/index.js

```diff
@@ -21,8 +20,8 @@ app.get("/", (req, res) => {
   ++pageVisitedTimesTotal;
```

Comment:
**praise:** don't use pre increment whenever it's not necessary action (From logic perspective), it's less readable than postincrement.